### PR TITLE
Add customers resource

### PR DESCRIPTION
**Pull request by criskell**
This PR addresses #3

### CODE REVIEW

File: tests/RequestTest.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    }
+
+    public function testShouldApplyPagination(): void
+    {
+        $streamFactory = $this->createMock(StreamFactoryInterface::class);
+        $requestFactory = $this->createMock(RequestFactoryInterface::class);
+
+        $requestFactory->expects($this->once())
+            ->method("createRequest")
+            ->with("GET", "https://example.com/costumer?skip=100&limit=200");
```
Comment (edumaciel10) [inline tests/RequestTest.php:70]: You are confusing customer with costumer

**Review by edumaciel10 — CHANGES_REQUESTED** [review]
Fix the customer naming

File: src/ApiErrorException.php [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace Openpix\PhpSdk;
```
Comment (daniloab) [inline src/ApiErrorException.php:3]: use in all places `OpenPix`

**Review by daniloab — COMMENTED** [review]
you can use this repo as inspiration https://github.com/DanielHe4rt/larapix/ since it is an implementation of our api with php

**Pull request merged**